.github/workflows/deployment-stage-prod.yml

```diff
@@ -86,6 +86,8 @@ jobs:
             --set php.oauth.jubladb.baseUrl='${{ secrets.OAUTH_JUBLADB_BASE_URL }}' \
             --set frontend.sentryDsn='${{ secrets.FRONTEND_SENTRY_DSN }}' \
             --set print.sentryDsn='${{ secrets.PRINT_SENTRY_DSN }}' \
+            --set print.renderHTMLTimeoutMs='${{ env.PRINT_RENDER_HTML_TIMEOUT_MS }}' \
```

Comment:
Shouldn't this be vars instead of env?

Do we actually intend to configure the timeouts differently for the various environments? If not, I'd also be ok to skip these 2 lines to keep the deployment command simpler.

Reply:
Not sure what you mean by "vars". If you meant default values in values.yaml, I would agree.

Reply:
yes, they are vars, my bad.

I would like to be able to verify this before deploying to stage.
(and i think it would be better to unify the 2 deployment workflows, that for the configurable values it is always clear what you have to do.
Then you don't have to think where i make what configurable, but that you make it configurable and always in the same way.
It is important that everything works if the vars or the secret is not available if this is possible.)
In my experience you don't know in advance what should be configurable.

Reply:
> In my experience you don't know in advance what should be configurable.

Not sure I got this last point. All parameters in `values.yaml` are configurable and hence `values.yaml` serves as the documentation what can be configured via `helm` command (for example for manual deployments).

I would object that we need to replicate all parameters which are theoretically configurable in our deployment scripts (they would get really really large; values.yaml has more than 200 lines at the moment). So unless we actually intend to have different values between dev/stage/prod I wouldn't list them in the deployment scripts.

If the intention is to play around with different values I'm ok. We can have them in and maybe remove later if we find good default values.